content/2_Bronze/Intro_DS.mdx

```diff
@@ -160,6 +160,18 @@ dynamic arrays (e.g. `vector<vector<int>>`) static arrays of dynamic arrays
 `vector<array<int,5>>`), and so on.
 
 <!-- Usually, the choice between a static array and a dynamic array is just personal preference. -->
+	
+An alternative for variable-length arrays are "doubly-linked lists." They are implemented by "linking" or pointing each element to the element before and after it. We recommend you look at [this Wikipedia article](https://en.wikipedia.org/wiki/Doubly_linked_list) for more information. These support O(1) insertion and removal, if you have access to the pointer of the element before hand.
+	
+<Warning>
+	Doubly-Linked lists cannot be accessed by indexes. This is a major reason why they are not used, as opposed to vectors or regular arrays
+</Warning>
+	
+The C++ object for a doubly-linked list is the `list` object.
+
+```cpp
+list<int> li
```

Comment:
does this only exist in c++
even if it does wrap it in a `<LanguageSection>`

Reply:
The entire thing is in a CPP Section

Reply:
oh sorry, the git diff is kinda weird

Reply:
```suggestion
list<int> li;
```